cookbook/attributes/default.rb

```diff
@@ -25,6 +25,12 @@
 default['redisio']['package_install'] =
   Chef::VersionConstraint.new('>= 14.04').include?(node['platform_version'])
 
+# Set to true if redis-server should be installed
+default['redis']['local'] = true
+
+# Set to true if mysql should be installed
+default['database']['local'] = true
+
```

Comment:
That comment is old and should probably be dumped.

Defaulting to local makes sense based on the cookbook's description in the README.
> This recipe will build a stand-alone instance fo the Guardian stack, including databases.

If users include this cookbook in a larger cookbook and want to specify remote db creds they can override that key.